### ISSUE TITLE
STEGO and SmooSeg comparison  on Potsdam-3

### ISSUE THREAD

**Issue by kuerbiskopf** [mc-lan/SmooSeg#5]
Hey, as someone who now trained and evaluated both STEGO and SmooSeg, I think the original STEGO authors trained and evaluated Potsdam-3 on Vit-small, or am I crazy? Even their provided pre-trained model is much smaller, whereas the other models for other datasets are larger and have "vit_base" in their name.

**Comment by kuerbiskopf** [comment]
I found this:
The training configuration for the Potsdam dataset differs from the other ones, i.e. using a smaller backbone, fewer training steps, no five-cropping of training images, and a smaller batch size.
From this paper: https://arxiv.org/abs/2304.07314
So it does seem STEGO uses Vit-S for Potsdam, but the authors of SmooSeg assumed they use ViT-B in their paper.

**Comment by mc-lan** [comment]
kindly refer to A.10 IMPLEMENTATION DETAILS in STEGO. _"STEGO uses the “ViT-Base” architecture of DINO pre-trained on ImageNet."_

**Comment by kuerbiskopf** [comment]
If I load the pretrained model of Potsdam from the STEGO authors and load it into eval_segmentation and print out the model.cfg, I get: 
model_type: vit_small
arch: dino

When I do evaluation using that model on Potsdam, I get a slightly lower Accuracy of 75.8% - probably because I had to lower the batch size and only have one GPU available. That's rather close to the 77% the authors of STEGO claim to achieve.

**Comment by mc-lan** [comment]
The batch size is not relacant to the evaluation result. One possible guess is that ViT-S has similar performance to ViT-B on Potsdam-3, given that it contains only three categories.
